src/common/tvgMath.h

```diff
@@ -386,6 +386,50 @@ static inline void log(const Point& pt)
 }
 
 
+static inline bool shouldMergePts(const Point& p1, const Point& p2, float tolerance)
```

Comment:
merge is too specific purpose. I think we can make the function more generic like:
-> `bool closed(p1, p2, tolerance)`